templates/zerver/help/upload_share_files.md

```diff
@@ -0,0 +1,26 @@
+## Upload and Share Files
+
+The paperclip icon (![paperclip](/static/images/help/paperclip.png)) under the message field lets you attach files, including images, sound, and video. Alternatively, you may also drag and drop files on the message field. The files are uploaded to a server and a link is supplied, but a thumbnail is displayed if possible. You’ll see the link in Markdown format: `[link_text](link_URL)`
+
+The image below demonstrates what the Markdown format will look like when you upload your file.
+![before](/static/images/help/file_upload_before.png)
+
+If you wish to give your image a caption or title, you may add text before or after the generated link. Any text you add in the box will not break the link to your file so long as the file link remains intact.
+
+---
+
+After submitting your message, you will see a thumbnail if your file is a common image type (i.e. PNG, JPEG, etc.). A file that is a common image type will have a thumbnail along with a link to the file as shown in the image below while all other files will simply have a link displayed.
```

Comment:
A little redundent. Maybe - 
After submitting your message, you will see a clickable thumbnail if your file is a common image type (i.e. PNG, JPEG, etc.). All other files will simply have a link displayed.